### PR TITLE
Updating common files

### PR DESCRIPTION
**Pull request by diehlaws**
*Description of changes:*
Updating common files (readme, code of conduct, contributing) to standardize content & formatting across all AWS SDKs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,14 +1,44 @@
-Contributing to the AWS SDK for Go
+# Contributing to the AWS SDK for Go
 
+Thank you for your interest in contributing to the AWS SDK for Go!
 We work hard to provide a high-quality and useful SDK, and we greatly value
 feedback and contributions from our community. Whether it's a bug report,
 new feature, correction, or additional documentation, we welcome your issues
 and pull requests. Please read through this document before submitting any
-issues or pull requests to ensure we have all the necessary information to
+[issues][] or [pull requests][pr] to ensure we have all the necessary information to
```
Comment (jasdel) [inline CONTRIBUTING.md:8]: Do these markdown links need to be filled in? looks like the `issues` one does?  I think you can also use just `[issues]` without the trailing `[]` to like to a reference of the same name.

File: README.md [hunk 2]
```diff
@@ -85,6 +96,31 @@ func main() {
 }
 ```
 
+## Getting Help
+
+Please use these community resources for getting help. We use the GitHub issues
+for tracking bugs and feature requests.
+
+* Ask a question on [StackOverflow](http://stackoverflow.com/) and tag it with the [`aws-sdk-go`](http://stackoverflow.com/questions/tagged/aws-sdk-go) tag.
+* Come join the AWS SDK for Go community chat on [gitter](https://gitter.im/aws/aws-sdk-go).
```
Comment (jasdel) [inline README.md:105]: Nit this is the v1 channel. there is an `aws-sdk-go-v2` channel as we.. But may make sense to use just one for more collaboration.

File: README.md [hunk 3]
```diff
@@ -95,16 +131,40 @@ The v2 SDK will use GitHub [Issues] to track feature requests and issues with th
 
 **Contributing**. You can open pull requests for fixes or additions to the AWS SDK for Go 2.0 Developer Preview release. All pull requests must be submitted under the Apache 2.0 license and will be reviewed by an SDK team member before being merged in. Accompanying unit tests, where possible, are appreciated.
 
-## License
+## Resources
+
+[Developer guide](https://docs.aws.amazon.com/sdk-for-go/v2/developer-guide/welcome.html/) - This document
```
Comment (jasdel) [inline README.md:136]: Nit this is a 404 page currently.

File: README.md [hunk 4]
```diff
@@ -95,16 +131,40 @@ The v2 SDK will use GitHub [Issues] to track feature requests and issues with th
...
+[SDK Examples](https://github.com/aws/aws-sdk-go-v2/tree/master/example) -
+Included in the SDK's repo are several hand crafted examples using the SDK
+features and AWS services.
+
+[Forum](https://forums.aws.amazon.com/forum.jspa?forumID=293) - Ask questions, get help, and give feedback
 
-This SDK is distributed under the
-[Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0),
-see LICENSE.txt and NOTICE.txt for more information.
+[Issues](https://github.com/aws/aws-sdk-go-v2/issues) - Report issues, submit pull requests, and get involved
```
Comment (jasdel) [inline README.md:161]: probably should use `[Issues][issues]` for these references that are already defined.

File: README.md [hunk 5]
```diff
@@ -85,6 +96,31 @@ func main() {
...
+* Come join the AWS SDK for Go community chat on [gitter](https://gitter.im/aws/aws-sdk-go).
+* Open a support ticket with [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html).
+* If you think you may have found a bug, please open an [issue](https://github.com/aws/aws-sdk-go-v2/issues/new/choose).
+
+This SDK implements AWS service APIs. For general issues regarding the AWS services and their limitations, you may also take a look at the [Amazon Web Services Discussion Forums](https://forums.aws.amazon.com/).
+
+### Opening Issues
+
+If you encounter a bug with the AWS SDK for Go we would like to hear about it.
+Search the [existing issues](https://github.com/aws/aws-sdk-go-v2/issues) and see
```
Comment (jasdel) [inline README.md:114]: nit probably better to use the `[existing issues][issues]` reference for these that are already defined.

File: CONTRIBUTING.md (hunk 1 above)
Comment (diehlaws) [reply CONTRIBUTING.md]: Link works as is but suggested formatting is cleaner. Fixed!

File: README.md (hunk 2 above)
Comment (diehlaws) [reply README.md]: Fixed! If we decide to merge the two channels later on we can update this link accordingly.

File: README.md (hunk 3 above)
Comment (diehlaws) [reply README.md]: Removed link until it's live.

File: README.md (hunk 5 above)
Comment (diehlaws) [reply README.md]: Fixed!

File: README.md (hunk 4 above)
Comment (diehlaws) [reply README.md]: Fixed!

**Pull request merged**